### PR TITLE
Adding support for nested quotes in attribute selection

### PR DESCRIPTION
**Pull request by himanshu-pro**
Tests and resolution added for the issue (#987) in org.jsoup.parser package

### CODE REVIEW

File: src/test/java/org/jsoup/parser/TokenQueueTest.java [hunk 1]
```diff
@@ -1,74 +1,98 @@
...
-		TokenQueue tq = new TokenQueue(t);
-		String data = tq.chompToIgnoreCase("</textarea>");
-		assertEquals("<textarea>one < two ", data);
-		
-		data = tq.chompToIgnoreCase("</textarea>");
-		assertEquals(" third ", data);
-    }
-    
+@RunWith(JUnitParamsRunner.class) public class TokenQueueTest {
+  @Test public void chompBalanced() {
```
Comment (cketti) [inline src/test/java/org/jsoup/parser/TokenQueueTest.java]: The rest of the code base uses 4 spaces for indentation. Did you change it here on purpose?

Comment (himanshu-pro) [reply src/test/java/org/jsoup/parser/TokenQueueTest.java]: No. Sorry I didn't know if you had strict guidelines for style. But I've been trying to move to Google style guide and used it here too. Would you like me to submit a different pull request with the original style.?

**Comment by himanshu-pro** [comment]
Any updates..? Thanks

**Comment by jhy** [comment]
Thanks, merged. I removed the junit params as I found them harder to read than just a few method calls.

**Pull request merged**